components/projectsView/ProjectFilters.js

```diff
@@ -76,16 +74,20 @@ export default function ProjectFilters({
     <Row className="text-center">
       {isTabletOrMobile ? (
         <Col key="projectStatusFilter">
-          <Row className={`bg-light h-100 `}>
+          <Row
+            className={`h-100`}
+            style={{
+              backgroundColor: "#f8f9fa",
```

Comment:
using `bg-light` as a className to change the background color isn't working anymore with the upgrade so I had to change the background color with styling

Reply:
i think what's going on here is that the Bootstrap scss customizations changed a bit with v5. we are overriding `$theme-colors` is our `custom.scss` file. for example, `bg-primary` works, because it's defined there. so adding an entry for `light` should fix this!